src/routes/safe/container/hooks/useLoadSafe.tsx

```diff
@@ -15,7 +15,7 @@ export const useLoadSafe = (safeAddress) => {
   useEffect(() => {
     const fetchData = () => {
       if (safeAddress) {
-        dispatch(fetchLatestMasterContractVersion())
+        dispatch<any>(fetchLatestMasterContractVersion())
```

Comment:
Is possible to use `unknown` here?

Reply:
no, but there's a fix that Mikhail did around dispatch. I'll fix this.